doc/sphinx/src/using-closures.rst

```diff
@@ -3,126 +3,257 @@
+essence the PTE solver has the form
 
 .. math::
 
-  P = P_0 = P_1 = \ldots = P_{N - 1}
+  P_i = F(\epsilon, f_\mathrm{tot}, f_1, ..., f_i, ..., f_{N-1},
+          \rho_1, ..., \rho_i, ..., \rho_{N-1}) \\
+  \rho_i = G(\epsilon, f_\mathrm{tot}, f_1, ..., f_i, ..., f_{N-1},
+          \rho_1, ..., \rho_i, ..., \rho_{N-1}) \\
+  \epsilon_i = H(\epsilon, f_\mathrm{tot}, f_1, ..., f_i, ..., f_{N-1},
+          \rho_1, ..., \rho_i, ..., \rho_{N-1}) \\
+  T_i = J(\epsilon, f_\mathrm{tot}, f_1, ..., f_i, ..., f_{N-1},
+          \rho_1, ..., \rho_i, ..., \rho_{N-1})
+
+
+The important nuance here is the **the volume fractions and densities are both
+inputs and outputs** in the current ``singularity-eos`` formulation of the
+closure models. From physics perspective this can be confusing, but from a code
+perspective this limits the number of variables that need to be passed to the
+PTE solver and provides a convenient way to specify an initial guess for the
+closure state.
+
+.. warning::
+
+  The volume fractions and material densities must be consistent in some way
+
+.. note::
+
+  The mass fraction information is encoded in the specifciation of :math:`f_i`
+  and :math:`\rho_i`. In order to convert to component densities,
+  :math:`\rho_i` from mass fractions, :math:`\mu_i` and total density,
+  :math:`\rho`, volume fractions must be ***assumed*** in some consistent way
+  so that
+
+  .. math::
+
+    \rho_i = \frac{\mu_i \rho}{f_i}
+
+  .. math::
+
+    \sum\limits_{i=0}^{N-1} f_i = f_\mathrm{tot}.
+
+  One such assumption would be to **equipartition** the volume between all
+  materials such that
+
+  .. math::
+
+    f_i = \frac{1}{N}
+
+Pressure-Temperature Equilibirum
+================================
 
-where each pressure is computed as either
+At present, ``singularity-eos`` focuses on several methods for finding a PTE
+solution, i.e. one where the pressures and temperatures of the individual
+materials are all the same. The methods presented differ in what they treat as
+independent variables, and thus what precise system of equations they solve.
+However they all share the above mathematicaly formulation.
+
+In essence, the PTE equations can be posed as two residual equations:
+
+.. math::
+
+  f_\mathrm{tot} - \sum\limits_{i=0}^{N-1} f_i = 
+    \sum\limits_{i=0}^{N-1} f_i^*(x_i*, y_i*) - f_i(x_i, y_i)
+
+.. math::
+
+  u_\mathrm{tot} - \sum\limits_{i=0}^{N-1} u_i = 
+    \sum\limits_{i=0}^{N-1} u_i^*(x_i*, y_i*) - u_i(x_i, y_i)
+
+where the superscript :math:`^*` denotes the variables at the PTE state,
+:math:`f` corresponds to the volume fractions, and :math:`u` to the energy
+density (see the previous section for more information). In these equations,
+:math:`x` and :math:`y` represent some choice of independent thermodynamic
+variables.
+
+Then the energy and volume fraction constraint equations can be Taylor-expanded
+about the equilibrium states :math:`f_i^*(\rho_i, y_i)` and
+:math:`u_i^*(\rho_i, y_i)` so that they become
```

Comment:
I dunno if I agree that we are assuming a Taylor expansion. The Taylor expansion is used by the Newton-Raphson solve to compute local gradients, but we are solving for the root of a function generically. The linear solve is a *piece* of the generic solve.

Reply:
I don't really understand your distinction. My understanding of a Newton solve is that it is derived from the Taylor expansion of the residual function about the root.

Reply:
I guess I'm concerned that this implies that the *global* root is computed using the *local* taylor expansion. However, that's not the case. We re-compute the taylor expansion at each iteration of the outer loop of the solve, so that the global solution is not limited to the local tangent space.

Reply:
Ahhhh I see. Okay I'll try to reword things to better reflect that we are still solving the fully-non-linear system and that the non-linear solve is built off of the linearization via Taylor expansion.

Reply:
@Yurlungur see if you like my new wording. I tried to highlight more clearly that this step is needed for the linearization of the problem in preparation for a Newton-Raphson method.  (docs may take a bit to build)